coresimd/src/x86/i686/sse.rs

```diff
@@ -217,57 +215,57 @@ pub unsafe fn _mm_cvtpi32_ps(a: f32x4, b: i32x2) -> f32x4 {
 #[inline(always)]
 #[target_feature = "+sse"]
 #[cfg_attr(test, assert_instr(cvtpi2ps))]
-pub unsafe fn _mm_cvt_pi2ps(a: f32x4, b: i32x2) -> f32x4 {
+pub unsafe fn _mm_cvt_pi2ps(a: __m128, b: i32x2) -> __m128 {
```

Comment:
Why is `i32x2` still used here? Or is this only a partial migration of the `i686::sse` module?

Reply:
It's the `i586::sse` module that was migrated, not the `i686::sse` one.

Reply:
Ah yeah oops those are some typos in the PR title/description. I'll be fixing up the other modules as I go along (those one was focused on i586)